pkg/frontend/v1/frontend.go

```diff
@@ -83,14 +84,18 @@ func New(cfg Config, limits Limits, log log.Logger, registerer prometheus.Regist
 			Name: "cortex_query_frontend_queue_length",
 			Help: "Number of queries in the queue.",
 		}, []string{"user"}),
+		discardedQueries: promauto.With(registerer).NewCounterVec(prometheus.CounterOpts{
```

Comment:
Please cleanup this metric in `cleanupInactiveUserMetrics` method.

Reply:
👍